src/shared/discovery.js

```diff
@@ -2,7 +2,7 @@
  * Callback invoked when another frame is discovered in this window which runs
  * the Hypothesis sidebar or annotation layer code.
  *
- * @type {Function} DiscoveryCallback
+ * @callback DiscoveryCallback
```

Comment:
could this be `@class` too? I'm just trying to understand if this is simply a better vs best or right vs wrong?

Reply:
`@type {Function}` isn't legal syntax for declaring a function/callback type with its parameters and return value. I think `@typedef {Function} DiscoveryCallback` would also have worked here, but `@callback` is shorter.